recipe/meta.yaml

```diff
@@ -12,9 +12,9 @@ source:
 build:
     number: 1
     features:
-        - vc9                [win and py27]
-        - vc10               [win and py34]
-        - vc14               [win and py35]
+        - vc9   # [win and py27]
+        - vc10  # [win and py34]
+        - vc14  # [win and py35]
```

Comment:
:+1: It can have more than 2 spaces if you want. Just it needs at least 2. The rest is pretty much fixed though.